chore: updated tests to use specific asserts in magazine workshop steps 31-37

baibhavmandal opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or Gitpod.

Replaced generic assertions with the suggested specific assert methods across multiple test cases, without altering what is being tested.

Closes #61240

baibhavmandal committed:
fix: updated tests to use specific asserts in magazine workshop stesp 31-37
#61240

majestic-owl448 reviewed (changes requested):
I have updated your post to include the checklist, please make sure to do the things listed and check them off.

Also please update the title to include step numbers and workshop name.

The changes itself are good, so if you can tidy up the PR we can merge it 👍

baibhavmandal commented:
Hi! Thanks so much for your support and guidance. I've updated the PR title to include the step numbers and workshop name, and added the checklist as requested. Would appreciate it if you could take a look.

majestic-owl448 reviewed (approved):
Hi @baibhavmandal . Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝
